spec/jobs/notify_compatible_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe NotifyCompatiblesJob, type: :job do
+  let(:user) { create(:user) }
+  let!(:target) { create(:target, user: user, latitude: 10.00, longitude: 11.00, topic: 'art',  length: 500000) }
+  let(:notify_compatible) { NotifyCompatiblesJob.new(target) }
+  
+  context 'whene there are no targets compatibles' do
+    it 'does not notify user' do
+      ActiveJob::Base.queue_adapter = :test
```

Comment:
Move this configuration to `rails_helper` file, otherwise you'll have to include it in all the examples.